schema-source/RSxxSub.schema.yaml

```diff
@@ -0,0 +1,224 @@
+    incidence_angle_azimuth:
+      Description: "Incidence angle azimuth"
+      Data Type: "[Numeric][1..]"
+      Required: if is_incidence_hemispherical=FALSE
+      Units: "radians"
+      Constraints: [">=0.0", "<=6.28319"]
+      Notes: "The direction of incidence is projected onto the plane of the surface of the sample. The azimuth is the angle between this projection and the prime direction. It can have values between 0� and 360�. If we look at the sample as if it was a clock, the prime direction being 12 o'clock and the projection 11 o'clock, then the azimuth equals 30� because of the right-handed coordinate system of [ISO 80000-2:2019, Quantities and units - Part 2: Mathematics](https://www.iso.org/standard/64973.html)."
+    emergence_angle_polar:
+      Description: "Emergence angle polar"
+      Data Type: "[Numeric][1..]"
+      Required: if emergence_direction=SPECIFIC_DIRECTION
+      Units: "radians"
+      Constraints: [">=0.0", "<=1.5708"]
+      Notes: "The polar angle is the angle between the direction of emergence and an axis which is perpendicular to the surface of the sample. It can have values between 0� and 90�. A polar angle of 0� means that the direction of emergence is perpendicular to the sample. A polar angle of 90� means that the direction of emergence is parallel to the sample. "
+    emergence_angle_azimuth:
+      Description: "Emergence angle azimuth"
+      Data Type: "[Numeric][1..]"
+      Required: if emergence_direction=SPECIFIC_DIRECTION
+      Units: "radians"
+      Constraints: [">=0.0", "<=6.28319"]
+      Notes: "The direction of emergence is projected to the plane of the surface of the sample. The azimuth is the angle between this projection and the prime direction. It can have values between 0� and 360�. If we look at the sample as if it was a clock, the prime direction being 12 o'clock and the projection 11 o'clock, then the azimuth equals 30� because of the right-handed coordinate system of [ISO 80000-2:2019, Quantities and units - Part 2: Mathematics](https://www.iso.org/standard/64973.html)."
+    thermochromic_temperature:
+      Description: "Temperature used for modulating thermochromic glazing substrates"
+      Data Type: "[Numeric][1..]"
+      Units: "K"
+      Constraints: ">=0.0"
+      Required: False
+    photochromic_illuminance:
+      Description: "Illuminance used for modulating photochromic glazing substrates"
+      Data Type: "[Numeric][1..]"
+      Units: "lux"
+      Constraints: ">=0.0"
+      Required: False
+    electrochromic_voltage:
+      Description: "Voltage used for modulating electrochromic glazing substrates"
+      Data Type: "[Numeric][1..]"
+      Units: "V"
+      Constraints: ">=0.0"
+      Required: False
+
+LookupVariablesOptical:
+  Object Type: "Lookup Variables"
+  Data Elements:
+    transmittance:
```

Comment:
There are components for which front_side_transmittance does not equal back_side_transmittance .